[2023-12-10 19:31:05,099] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-09-04T00:00:00+00:00 [queued]>
[2023-12-10 19:31:05,218] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-09-04T00:00:00+00:00 [queued]>
[2023-12-10 19:31:05,221] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:31:05,225] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:31:05,236] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:31:05,393] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-09-04 00:00:00+00:00
[2023-12-10 19:31:05,425] {standard_task_runner.py:52} INFO - Started process 552 to run task
[2023-12-10 19:31:05,432] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-09-04T00:00:00+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpzeh7ym26', '--error-file', '/tmp/tmpghw1am8v']
[2023-12-10 19:31:05,476] {standard_task_runner.py:80} INFO - Job 107: Subtask segunda_tarea
[2023-12-10 19:31:06,059] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-09-04T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:31:06,756] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-09-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-04T00:00:00+00:00
[2023-12-10 19:31:06,760] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:31:06,761] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:31:06,830] {subprocess.py:85} INFO - Output:
[2023-12-10 19:31:06,835] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:31:06,836] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:31:07,105] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20220904T000000, start_date=20231210T193105, end_date=20231210T193107
[2023-12-10 19:31:07,439] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:31:08,145] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
